CHANGELOG.md

```diff
@@ -14,9 +13,8 @@
 * File format: generates Realms with format v23 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 or later for synced Realms).
 
 ### Internal
-<!-- * Either mention core version or upgrade -->
-<!-- * Using Realm Core vX.Y.Z -->
-<!-- * Upgraded Realm Core from vX.Y.Z to vA.B.C -->
+* Upgraded Realm Core from v13.1.1 to v13.1.2. ([#5174](https://github.com/realm/realm-js/issues/5174))
+* Migrated to the new `Subscription` API in Realm Core.
```

Comment:
This has no real impact on users and was a small code change so I don't know if it is worth calling out in release notes. Your call.

Reply:
I'll remove it.